style.css

```diff
@@ -387,6 +390,82 @@ hr {
   text-align: center;
 }
 
```

Comment:
### CARDS

**[OPTIONAL]**

- Great job with your changes, however, I think these sections should look the same as the vertical layout in your desktop version, while you've added a breakpoint that makes it look like the main design, kindly make it the same on all other desktop screen sizes.


*Update this as seen in a bigger screen*
![Screenshot from 2023-05-22 17-54-39](https://github.com/aradradev/first-capstone-project/assets/59397159/c9bfbfe6-c9e7-4eb7-ae58-409db7c991fa)

*to this also in bigger screen*
![Screenshot from 2023-05-22 17-54-26](https://github.com/aradradev/first-capstone-project/assets/59397159/e61e8381-d52a-46cf-a3b7-4f5ec37b50a5)